components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8360,7 +8365,7 @@ def getZoomLevelScaling(self):
         return rv
 
     def setActiveChannels(self, channels, windows=None, colors=None,
-                          reverseMaps=None):
+                          invertMaps=None):
```

Comment:
This change counts as a breaking change. Since the rest of this PR attempts to be backwards compatible, it makes sense to do so here as well. `if reverseMaps is not None` could issue a warning so that it's effectively deprecated.